### ISSUE TITLE
dbeaver-ce v5.2.5 snap fails on launch exec

### ISSUE THREAD

**Issue by pgnd** [dbeaver/dbeaver#4660]
```
snap --version
	snap           2.36.1-lp150.2.1
	snapd          2.36.1-lp150.2.1
	series         16
	opensuse-leap  15.0
	kernel         4.19.2-lp150.3.g21d18c9-default

snap search dbeaver
	Name        Version  Publisher      Notes  Summary
	dbeaver-ce  5.2.5    rasul-dbeaver  -      Universal Database Tool

cref: https://snapcraft.io/dbeaver-ce

snap install dbeaver-ce --edge
snap list dbeaver-ce
	Name        Version  Rev  Tracking  Publisher      Notes
	dbeaver-ce  5.2.5    30   edge      rasul-dbeaver  -

JAVA_HOME=/usr/lib64/jvm/java-1.8.0-openjdk-1.8.0/ /snap/bin/dbeaver-ce.dbeaver

	Dbeaver:
	An error has occurred. See the log file
	/home/dev/.eclipse/1738816240_linux_gtk_x86_64/configuration/1542773375172.log.

cat /home/dev/.eclipse/1738816240_linux_gtk_x86_64/configuration/1542773375172.log

	!SESSION 2018-11-20 20:09:29.657 -----------------------------------------------
	eclipse.buildId=unknown
	java.version=9-internal
	java.vendor=Oracle Corporation
	BootLoader constants: OS=linux, ARCH=x86_64, WS=gtk, NL=en_US

	!ENTRY org.eclipse.osgi 4 0 2018-11-20 20:09:35.481
	!MESSAGE Error determining system packages.
	!STACK 0
	java.lang.ClassNotFoundException: java.lang.ModuleLayer
	        at java.net.URLClassLoader.findClass(java.base@9-internal/URLClassLoader.java:384)
	        at java.lang.ClassLoader.loadClass(java.base@9-internal/ClassLoader.java:486)
	        at java.lang.ClassLoader.loadClass(java.base@9-internal/ClassLoader.java:419)
	        at java.lang.Class.forName0(java.base@9-internal/Native Method)
	        at java.lang.Class.forName(java.base@9-internal/Class.java:294)
	        at org.eclipse.osgi.storage.Storage.calculateVMPackages(Storage.java:1714)
	        at org.eclipse.osgi.storage.Storage.calculateVMProfile(Storage.java:1654)
	        at org.eclipse.osgi.storage.Storage.findVMProfile(Storage.java:1552)
	        at org.eclipse.osgi.storage.Storage.loadVMProfile(Storage.java:1509)
	        at org.eclipse.osgi.storage.Storage.getBuilder(Storage.java:766)
	        at org.eclipse.osgi.storage.Storage.checkSystemBundle(Storage.java:394)
	        at org.eclipse.osgi.storage.Storage.createStorage(Storage.java:177)
	        at org.eclipse.osgi.internal.framework.EquinoxContainer.<init>(EquinoxContainer.java:72)
	        at org.eclipse.osgi.launch.Equinox.<init>(Equinox.java:34)
	        at org.eclipse.core.runtime.adaptor.EclipseStarter.startup(EclipseStarter.java:306)
	        at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:242)
	        at sun.reflect.NativeMethodAccessorImpl.invoke0(java.base@9-internal/Native Method)
	        at sun.reflect.NativeMethodAccessorImpl.invoke(java.base@9-internal/NativeMethodAccessorImpl.java:62)
	        at sun.reflect.DelegatingMethodAccessorImpl.invoke(java.base@9-internal/DelegatingMethodAccessorImpl.java:43)
	        at java.lang.reflect.Method.invoke(java.base@9-internal/Method.java:531)
	        at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:659)
	        at org.eclipse.equinox.launcher.Main.basicRun(Main.java:595)
	        at org.eclipse.equinox.launcher.Main.run(Main.java:1501)
	        at org.eclipse.equinox.launcher.Main.main(Main.java:1474)

	!ENTRY org.eclipse.osgi 4 0 2018-11-20 20:09:37.670
	!MESSAGE Application error
	!STACK 1
	java.lang.UnsatisfiedLinkError: Could not load SWT library. Reasons:
	        /home/dev/.eclipse/1738816240_linux_gtk_x86_64/configuration/org.eclipse.osgi/146/0/.cp/libswt-pi3-gtk-4919.so: libgtk-3.so.0: cannot open shared object file: No such file or directory
	        no swt-pi3-gtk in java.library.path
	        /home/dev/.swt/lib/linux/x86_64/libswt-pi3-gtk-4919.so: libgtk-3.so.0: cannot open shared object file: No such file or directory
	        Can't load library: /home/dev/.swt/lib/linux/x86_64/libswt-pi3-gtk.so

	        at org.eclipse.swt.internal.Library.loadLibrary(Library.java:330)
	        at org.eclipse.swt.internal.Library.loadLibrary(Library.java:242)
	        at org.eclipse.swt.internal.gtk.OS.<clinit>(OS.java:83)
	        at org.eclipse.swt.internal.Converter.wcsToMbcs(Converter.java:209)
	        at org.eclipse.swt.internal.Converter.wcsToMbcs(Converter.java:155)
	        at org.eclipse.swt.widgets.Display.<clinit>(Display.java:151)
	        at org.jkiss.dbeaver.core.application.DBeaverApplication.start(DBeaverApplication.java:105)
	        at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:199)
	        at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:137)
	        at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:107)
	        at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:391)
	        at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:246)
	        at sun.reflect.NativeMethodAccessorImpl.invoke0(java.base@9-internal/Native Method)
	        at sun.reflect.NativeMethodAccessorImpl.invoke(java.base@9-internal/NativeMethodAccessorImpl.java:62)
	        at sun.reflect.DelegatingMethodAccessorImpl.invoke(java.base@9-internal/DelegatingMethodAccessorImpl.java:43)
	        at java.lang.reflect.Method.invoke(java.base@9-internal/Method.java:531)
	        at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:659)
	        at org.eclipse.equinox.launcher.Main.basicRun(Main.java:595)
	        at org.eclipse.equinox.launcher.Main.run(Main.java:1501)
	        at org.eclipse.equinox.launcher.Main.main(Main.java:1474)

ls -al /home/dev/.swt/lib/linux/x86_64/
	-rwxr-xr-x 1 dev users 475K Nov 20 20:01 libswt-pi3-gtk-4919.so*
	-rwxr-xr-x 1 dev users 451K Nov 20 19:59 libswt-pi-gtk-4919.so*

```

**Comment by serge-rider** [comment]
Snap version is not eleased yet (we are working on it in #2299)

**Comment by uslss** [comment]
will be implemented in scope of #2299
